server/lib/budget.js

```diff
@@ -374,45 +374,42 @@ export async function getYearlyIncome(collective) {
   // TODO: support netAmountInHostCurrency
   const result = await sequelize.query(
     `
-      WITH "activeMonthlySubscriptions" as (
-        SELECT d."SubscriptionId", ROUND(AVG(t."netAmountInCollectiveCurrency")) AS "netAmountInCollectiveCurrency"
-        FROM "Transactions" t
-        LEFT JOIN "Orders" d ON d.id = t."OrderId"
-        LEFT JOIN "Subscriptions" s ON s.id = d."SubscriptionId"
-        WHERE t."CollectiveId"=:CollectiveId
-          AND t."RefundTransactionId" IS NULL
+      SELECT
+        (
+          SELECT COALESCE(SUM(o."totalAmount"), 0) * 12
+          FROM "Orders" o
+          INNER JOIN "Subscriptions" s ON o."SubscriptionId" = s.id
+          WHERE o."CollectiveId" = :CollectiveId
+          AND o."deletedAt" IS NULL
+          AND s."deletedAt" IS NULL
           AND s."isActive" IS TRUE
           AND s.interval = 'month'
-          AND s."deletedAt" IS NULL
-        GROUP BY d."SubscriptionId"
-      )
-      SELECT
-        (SELECT
-          COALESCE(SUM("netAmountInCollectiveCurrency"*12),0) FROM "activeMonthlySubscriptions")
+        )
         +
-        (SELECT
-          COALESCE(SUM(t."netAmountInCollectiveCurrency"),0) FROM "Transactions" t
-          LEFT JOIN "Orders" d ON t."OrderId" = d.id
-          LEFT JOIN "Subscriptions" s ON d."SubscriptionId" = s.id
-          WHERE t."CollectiveId" = :CollectiveId
-            AND t."RefundTransactionId" IS NULL
-            AND CAST(("t"."data"#>>'{internal}') AS BOOLEAN) IS NOT true
```

Comment:
We agreed that it was ok to try to stop supporting this property. Because it's not actively used since around 1 year. We will re-introduced if needed.